### PR TITLE
[QGDS-5] Establish editor environment and code formatting configs

### PR DESCRIPTION
**Pull request by stvp-qld**
This PR implements part of [QGDS-5] to establish consistent coding environment for multiple devs.

- Adds prettier library, dependencies and config for IDE formatting
- Adds a .vscode config to set default formatting behaviours 
- Updates eslint config to use the prettier config for formatting
 

[QGDS-5]: https://ssq-qol.atlassian.net/browse/QGDS-5?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: chromatic.config.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "projectId": "Project:66c421d9ea5f69287ba33241",
```
Comment (duttonw) [inline chromatic.config.json:2]: please remove, this locks this to qld-gov-au chromatic instead of anyones forked version.

File: package.json
Comment (duttonw) [inline package.json:1]: please add script for linting into package.json and update aws github actions to run the linting plugin.

**Review by duttonw — CHANGES_REQUESTED** [review]
looks good but needs to be also available on the command line for those who don't use vscode.

**Comment by duttonw** [comment]
https://github.com/qld-gov-au/qgds-vanilla/pull/172 should improve this pr.

File: chromatic.config.json (hunk 1 above)
Comment (stvp-qld) [reply chromatic.config.json:2]: chromatic.config.js added to .gitignore

File: package.json
Comment (stvp-qld) [reply package.json:1]: Thanks Will - I've merged your PR for eslint 9

**Comment by stvp-qld** [comment]
Merged to get other dev's up and running with common configs.  Will investigate Build and Test action fails.

**Pull request merged**